x-pack/plugins/fleet/public/applications/fleet/sections/agents/agent_list_page/components/agent_activity_button.tsx

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiButtonEmpty, EuiText, EuiTourStep } from '@elastic/eui';
+import React, { useEffect, useState } from 'react';
+import { FormattedMessage } from '@kbn/i18n-react';
+
+import { useStartServices } from '../../../../hooks';
+
+export const AgentActivityButton: React.FC<{
+  onClickAgentActivity: () => void;
+  showAgentActivityTour: { isOpen: boolean };
+}> = ({ onClickAgentActivity, showAgentActivityTour }) => {
+  const { uiSettings } = useStartServices();
+
+  const [agentActivityTourState, setAgentActivityTourState] = useState(showAgentActivityTour);
+
+  const hideTour = () => uiSettings.get('hideAgentActivityTour', false);
```

Comment:
should it be `uiSettings.set` instead of `.get`?

Reply:
oh I see it's a getter, why not getting the variable directly? instead of creating a function?

Reply:
probably it is enough to query directly, I found a similar usage here: https://github.com/elastic/kibana/blob/main/x-pack/plugins/fleet/public/components/add_agent_help_popover.tsx#L34